pkg/codec/vpx/vpx_image.go

```diff
@@ -0,0 +1,40 @@
+package vpx
+
+/*
+#cgo pkg-config: vpx
+#include <vpx/vpx_image.h>
+*/
+import "C"
+import "unsafe"
+
+type VpxImage struct {
```

Comment:
Users don't have a safe way to release the data.
Also to make the `Decoder` interface unified among the codecs, the decoder should return `image.Image`.

`mediadevices/io/video.Reader` returns release function along with `image.Image`.
So, the image implementing `image.Image` may have `C.vpx_image_t` internally and have release function written in CGO.


BTW, in Go's naming convention, it's `VPXImage`. https://go.dev/wiki/CodeReviewComments#initialisms